doc/CONFIGURATION.md

```diff
@@ -80,6 +81,23 @@ Here is an example least-privilege policy document to add to an IAM user or role
 
 Mountpoint also respects access control lists (ACLs) applied to objects in your S3 bucket, but does not allow you to automatically attach ACLs to objects created with Mountpoint. A majority of modern use cases in Amazon S3 no longer require the use of ACLs. We recommend that you keep ACLs disabled for your S3 bucket, and instead use bucket policies to control access to your objects.
 
+## Caching configuration
+
+Mountpoint now offers caching of metadata and object content allowing for reduced requests when reading files.
```

Comment:
Documentation is independent of time, so no "now".

```suggestion
Mountpoint can optionally cache object metadata and content to reduce cost and improve performance for repeated reads to the same file.
```

Reply:
makes sense.

I was planning to put "Since v1.2.0, Mountpoint can optionally cache...", but replaced it with "now" as a placeholder. Exclude that?